components/scream/src/physics/shoc/shoc_functions.hpp

```diff
@@ -74,9 +73,9 @@ struct Functions
     SHOCInput() = default;
 
     // Grid spacing of host model in x direction [m]
-    view_1d<const Pack1d> host_dx;
+    view_1d<const Scalar> host_dx;
```

Comment:
This might be a silly question, but why do we sometimes call this data type a `Scalar` and other times a `Real`?

Reply:
Good question.. I'm not quite sure why the shoc `Functions` structure allows for a different scalar type that Real in scream_types..